MaxText/standalone_dataloader.py

```diff
@@ -50,20 +50,22 @@ def data_load_loop(config, state=None):
   jax.block_until_ready(example_batch)
   first_end = datetime.datetime.now()
   time_to_load_first_batch = first_end-start
-  max_logging.log(f"First step completed in {time_to_load_first_batch} seconds")
+  if jax.process_index() == 0:
+    max_logging.log(f"STANDALONE DATALOADER: First step completed in {time_to_load_first_batch} seconds, on host 0")
 
   for _ in np.arange(start_step+1, config.steps):
     example_batch = load_next_batch(data_iterator, example_batch, config, mesh)
 
   jax.block_until_ready(example_batch) # wait until the last batch is read
   end = datetime.datetime.now()
-  max_logging.log(f"{config.steps} batches loaded in {end-start} seconds, on host {jax.process_index()}")
+  if jax.process_index() == 0:
+    max_logging.log(f"STANDALONE DATALOADER: {config.steps} batches loaded in {end-start} seconds, on host 0")
   return state
 
 
 def main(argv: Sequence[str]) -> None:
   jax.config.update('jax_default_prng_impl', 'unsafe_rbg')
```

Comment:
If you're getting rid of flags, maybe get rid of this one.

Reply:
Done, thanks